tests/ut/ir/transforms/test_hoist_scope_local_allocs.py

```
@@ -0,0 +1,223 @@
+        def k(
+            self, x: pl.Tensor[[64], pl.FP32], scratch: pl.Out[pl.Tensor[[64], pl.FP32]]
+        ) -> pl.Tensor[[64], pl.FP32]:
+            t = pl.load(x, [0], [64])
+            scratch = pl.store(t, [0], scratch)
+            return scratch
+
+        @pl.function(type=pl.FunctionType.Orchestration)
+        def main(self, x: pl.Tensor[[64], pl.FP32]) -> pl.Tensor[[64], pl.FP32]:
+            with pl.manual_scope():
+                scratch = pl.create_tensor([64], dtype=pl.FP32)
+                a, a_tid = pl.submit(self.k, x, scratch)
```

Ruff (0.15.20)

[warning] Unpacked variable `a_tid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)